tests/exporters/exporters_utils.py

```diff
@@ -32,14 +32,34 @@
     },  # Failed for DistilBERT: https://github.com/aws-neuron/aws-neuron-sdk/issues/645
     "electra": "hf-internal-testing/tiny-random-ElectraModel",
     "flaubert": "hf-internal-testing/tiny-random-flaubert",
-    "mobilebert": "hf-internal-testing/tiny-random-MobileBertModel",
-    "mpnet": "hf-internal-testing/tiny-random-MPNetModel",
-    "roberta": "hf-internal-testing/tiny-random-RobertaModel",
+    "mobilebert": {
+        "hf-internal-testing/tiny-random-MobileBertModel": [
+            "masked-lm",
+            "sequence-classification",
+            "token-classification",
+            "question-answering",
+        ],
+    },
+    "mpnet": {
+        "hf-internal-testing/tiny-random-MPNetModel": [
+            "masked-lm",
+            "sequence-classification",
+            "token-classification",
+            "question-answering",
+        ],
+    },
+    "roberta": {
+        "hf-internal-testing/tiny-random-RobertaModel": [
+            "masked-lm",
+            "sequence-classification",
+            "token-classification",
+            "question-answering",
+        ],
+    },
```

Comment:
Why?
If the other tasks are not supported, we should make sure we do not register them as suppoted!

Reply:
They are supported, but their tests on the default task are flaky(they succeed when I launch them individually but sometimes fail when I launch all tests, I need to investigate further why they failed randomly). I will remove them from the registration for now until I figure out why.

Reply:
Hmmm, that's weird. Maybe some state is shared among them.

Reply:
This will be corrected by #33 !

Reply:
Nope, it's still there... Will do further check.